text/0141-linking-traces.md

```diff
@@ -0,0 +1,367 @@
+                  ],
+                  // span and trace linked to (from span.spanContext()): 
+                  "spanId": "6c71fc6b09b8b716", 
+                  "traceId": "627a2885119dcc8184fae7eef09438cb", 
+                }
+              ],
+              "droppedLinksCount": 0
+            },
+            {
+              "traceId": "627a2885119dcc8184fae7eef09438cb",
+              "spanId": "6c71fc6b09b8b716",
+              "parentSpanId": "13d4cb0933154c29",
+              "name": "span-links-txn",
+              "links": [],
+              "droppedLinksCount": 0
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+
+## Current Session Models in Sentry SDKs
+
+Sentry SDKs do actually send session data, in fact even two types of sessions. However, neither of the two sessions are used and associated with a trace. 
+
+- SDK error sessions: SDKs currently send a session that counts and describes the error status of such a session. Depending on the error, the session is marked as crashed, abnormal or healthy, which is information that powers our Release (Health) product. These sessions are fundamentally flawed though because (at least in Browser JS) they only last as long as the currently loaded page. Every soft or hard navigation causes the session to be ended and a new one to be started. 
+- Replay Session: Frontend SDKs persist a replayId in the browser's `sessionStorage` which (even though the name does not suggest it) more accurately models a session than the SDKs' error sessions. We cannot have a hard dependency on this session as Replay is an extra product by Sentry, meaning this replayId is not always reliably set. However, the model of persisting the replay id can be used as a blueprint for how we could persist the last traceId. 
+
+Upon decision from Leadership as well as from it being noted in Sentry's Goal Hierarchy, we will not associate spans or traces via any of these sessions. Instead, the sessions will stay as-is and we will link traces in a "linked list"-style approach as described in this RFC. 
+
+# Supporting Data
+
+Sentry users would like to get as much insight as possible when inspecting (errors or performance) issues. Right now, with the notable exception of Session Replay, we can only provide answers as to what happened before an error occurred up to the point when a trace was started. If we can link from the current to the previous trace, we can significantly widen the insight into the entire user journey, which is often crucial to understand why and how a specific issue occurred. 
+
+Given that almost half of events sent to Sentry can be attributed alone to events sent from Sentry frontend JavaScript or mobile SDKs, a significant portion of sent events would directly benefit from better linkage between events. Considering that frontend applications almost always have a backend counter events sent from backend SDKs would implicitly also benefit from a better tracing model. Wins all around!
```

Comment:
```suggestion
Given that almost half of the events sent to Sentry can be attributed solely to events sent from Sentry's frontend JavaScript or mobile SDKs, a significant portion of sent events would directly benefit from better linkage between events. Considering that frontend applications almost always have a backend, counter events sent from backend SDKs would implicitly also benefit from a better tracing model. Wins all around!
```
Some structure improvements for better readability :)
But I am not sure about the phrase "counter events" resp. "backend counter events"